openpype/hosts/maya/api/lib.py

```diff
@@ -3981,7 +3981,7 @@ def create_rig_animation_instance(nodes, context, namespace, log=None):
     )
     assert roots, "No root nodes in rig, this is a bug."
 
-    asset = legacy_io.Session["AVALON_ASSET"]
+    asset = get_current_asset_name()  # remove legacy_io call
```

Comment:
```suggestion
    asset = get_current_asset_name()
```

Dont think we need to comment here :)